configuration/configuration_test.go

```diff
@@ -208,15 +208,6 @@ func saneDefaults(conf *payloads.Configure) bool {
 		conf.Configure.Launcher.MemoryLimit == true)
 }
 
-func TestFillDefaults(t *testing.T) {
```

Comment:
@obedmr we can still keep that one, and rename it TestInitDefaults().
It would check that conf.InitDefaults() builds a sane default configuration structure.